src/status_im/utils/js_resources.cljs

```diff
@@ -12,9 +12,8 @@
 (def web3 (str "; if (typeof Web3 == 'undefined') {"
                (slurp "node_modules/web3/dist/web3.min.js")
                "}"))
-(defn web3-init [provider-address current-account-address network-id]
-  (str "var providerAddress = \"" provider-address "\";"
-       "var currentAccountAddress = \"" current-account-address "\";"
+(defn web3-init [current-account-address network-id]
```

Comment:
how is it related to PRs objective?

Reply:
there was a subscription for this provider-address. just to do not optimize it, because its not used,  i removed it in this PR